mount-daemon/controllers/clientmount_controller.go

```diff
@@ -125,15 +125,25 @@ func (r *ClientMountReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 		return ctrl.Result{}, nil
 	}
 
+	clientMount.Status.Error = nil
```

Comment:
You shouldn't have to do this.  Nil should be the zero value for a pointer in Go.

Reply:
clientMount.Status.Error may have been set in a previous reconcile() call.